test-data/unit/check-literal.test

```diff
@@ -3149,3 +3149,18 @@ err_code = -TWO
 if bool():
     err_code = -THREE
 [builtins fixtures/float.pyi]
+
+[case testAliasForEnumTypeAsLiteral]
```

Comment:
Could you please actually add a test with enum and literal (as originally reported in the issue)?